bin/cli.js

```diff
@@ -101,23 +101,26 @@
                     method: req.method,
                     headers: {
                         "Content-Length": req.headers["content-length"] || 0,
-                        "Content-Type": req.headers["content-type"],
-                        "Authorization": req.headers["authorization"]
+                        "Content-Type": req.headers["content-type"] || '',
+                        "Authorization": req.headers["authorization"] || ''
                     },
                     followAllRedirects: true,
-                    body: body,
+                    body: body || '',
                     jar: false,
-                    strictSSL: false
+                    strictSSL: false,
+                    rejectUnauthorized : false
                 };
 
                 try {
-                    request(options, function(err, response, data) {
+                    needle.request(options.method, options.url, options.body, options, function(err, response, body){
                         try {
                             var statusCode = (response ? response.statusCode : 500) || 500;
                             var headers = (response ? response.headers : {}) || {};
+                            var bodyString = JSON.stringify(body);
 
+                            bodyString = bodyString.replace(/<[/]/g, "<\\/");
```

Comment:
Same feedback as other PR - let's try to turn parsing off for responses rather than parsing back to String: https://www.npmjs.com/package/needle#response-options

Reply:
Changes have been done to add parse_response option and tests are passing with new changes